rexchain/core/connectors.py

```diff
@@ -0,0 +1,85 @@
+import base64
```

Comment:
Se ve bien.

CUando dices merkle hash, estamos hablando de hashear el merkle root?

Reply:
Estoy pensando cambiar el comment a esto
`merkle_hash : Must be a valid sha256 str, as the merkle root is`